snuba/datasets/storages/validation.py

```diff
@@ -0,0 +1,28 @@
+from typing import Set
```

Comment:
I know we were planning to have a dedicated place for configuration validation to happen. If that exists, I think this should go there.